---
title: ALLFLAGS - Global assessment of the abundance, diversity and activity of marine
  heterotrophic flagellates species
authors:
- Dr Jane Doe
date: '2016-12-30'
publishDate: '2024-08-14T09:35:58.206637Z'
publication_types:
- project
funding: 'MINECO'
summary: 'Start: 2016 End: 2020'
start: '2016'
end: '2020'
abstract: ' The perception that marine microbial life is extremely diverse is well grounded based upon molecular diversity surveys and the existence of a wide variety of cultured forms. This also applies to the unpigmented smallest eukaryotes in planktonic systems, the heterotrophic flagellates (HFs), which form a diverse collection of tiny flagellated cells that are important agents in prokaryotic mortality through grazing and crucial in nutrient remineralization. HFs came to light a few decades ago, when they were included in marine food webs models as the trophic link between prokaryotes and larger protists like ciliates or dinoflagellates. However, HF cells have received relatively little attention in oceanographic efforts developed up to now, due to two main factors: (1) the lack of automatized counting procedures since they are not easily quantifiable by flow cytometry and (2) a particularly severe culturing bias, by which they seem to be composed by a myriad of novel and uncultured species. As a result, further research is needed to explore which are the dominant species forming HF assemblages, and go beyond the only fragmented information currently available based in scattered molecular surveys. The ALLFLAGS project aims to take advantage of new analytic tools and recent extensive sampling datasets to perform, for the first time, a global study to identify the dominant HF species and better define their ecological relevance in the marine environment. As novel tools, we will develop an automatized microscopy routine for counting HF cells and fully exploit the potential of High-Throughput Sequencing (HTS), together with new bioinformatic developments for diversity studies. Afterwards, we will apply these tools to quantify HF assemblages and identify the dominant species in recently available sampling surveys. In particular, we will process the dataset obtained in a global sampling effort done in the major oceans during the Malaspina expedition, and the samples from a long-term temporal survey performed at the Blanes Bay Microbial Observatory (BBMO). Once detected the dominant HF species in these two extensive surveys, we will undertake a detailed analysis of their distribution and abundance, including the definition of their genetic variability related to the environmental context, and the establishment of their relative activity measuring specific grazing rates. Finally, we will perform comparative genomics to identify the gene basis for ecological adaptation of selected dominant HF species using already published genomes and those obtained in novel metagenomes constructed here. This analysis will be based on the detection of gene families in all genomes and placing a particular focus on viral signatures as proxies of mortality susceptibility. The use of this genomic-based approach to its full potential will allow a better understanding of HFs role in the global marine ecosystem and biogeochemical cycles, as well as to elucidate the question of whether or not cultured and uncultured HF species can be differentiated based on their gene content. The confluence of HTS surveys and extensive datasets opens the possibility for the first time to develop a global assessment of HF diversity, the identification of the dominant HF species, and the search for their ecological niche. '
---
