---
title: Biosynthetic potential of the global ocean microbiome
authors:
- Lucas Paoli
- Hans-Joachim Ruscheweyh
- Clarissa C Forneris
- Florian Hubrich
- Satria Kautsar
- Agneya Bhushan
- Alessandro Lotti
- Quentin Clayssen
- Guillem Salazar
- Alessio Milanese
- Charlotte I Carlström
- Chrysa Papadopoulou
- Daniel Gehrig
- Mikhail Karasikov
- Harun Mustafa
- Martin Larralde
- Laura M Carroll
- Pablo Sánchez
- Ahmed A Zayed
- Dylan R Cronin
- Silvia G. Acinas
- Peer Bork
- Chris Bowler
- Tom O Delmont
- Josep M. Gasol
- Alvar D Gossert
- André Kahles
- Matthew B Sullivan
- Patrick Wincker
- Georg Zeller
- Serina L Robinson
- Jörn Piel
- Shinichi Sunagawa
date: '2022-07-01'
publishDate: '2024-08-08T12:00:47.030274Z'
publication_types:
- article-journal
publication: '*Nature*'
doi: 10.1038/s41586-022-04862-3
abstract: Abstract  Natural microbial communities are phylogenetically and metabolically
  diverse. In addition to underexplored organismal groups 1 , this diversity encompasses
  a rich discovery potential for ecologically and biotechnologically relevant enzymes
  and biochemical compounds 2,3 . However, studying this diversity to identify genomic
  pathways for the synthesis of such compounds 4 and assigning them to their respective
  hosts remains challenging. The biosynthetic potential of microorganisms in the open
  ocean remains largely uncharted owing to limitations in the analysis of genome-resolved
  data at the global scale. Here we investigated the diversity and novelty of biosynthetic
  gene clusters in the ocean by integrating around 10,000 microbial genomes from cultivated
  and single cells with more than 25,000 newly reconstructed draft genomes from more
  than 1,000 seawater samples. These efforts revealed approximately 40,000 putative
  mostly new biosynthetic gene clusters, several of which were found in previously
  unsuspected phylogenetic groups. Among these groups, we identified a lineage rich
  in biosynthetic gene clusters (‘ Candidatus Eudoremicrobiaceae’) that belongs to
  an uncultivated bacterial phylum and includes some of the most biosynthetically
  diverse microorganisms in this environment. From these, we characterized the phospeptin
  and pythonamide pathways, revealing cases of unusual bioactive compound structure
  and enzymology, respectively. Together, this research demonstrates how microbiomics-driven
  strategies can enable the investigation of previously undescribed enzymes and natural
  products in underexplored microbial groups and environments.
links:
- name: URL
  url: https://www.nature.com/articles/s41586-022-04862-3
pages: '111-118'
volume: '607'
number: '7917'
---
