---
title: Prevalent genome streamlining and latitudinal divergence of planktonic bacteria
  in the surface ocean
authors:
- Brandon K. Swan
- Ben Tupper
- Alexander Sczyrba
- Federico M. Lauro
- Manuel Martinez-Garcia
- José M. González
- Haiwei Luo
- Jody J. Wright
- Zachary C. Landry
- Niels W. Hanson
- Brian P. Thompson
- Nicole J. Poulton
- Patrick Schwientek
- Silvia G Acinas
- Stephen J. Giovannoni
- Mary Ann Moran
- Steven J. Hallam
- Ricardo Cavicchioli
- Tanja Woyke
- Ramunas Stepanauskas
date: '2013-07-01'
publishDate: '2024-08-05T15:04:47.351095Z'
publication_types:
- article-journal
publication: '*Proceedings of the National Academy of Sciences*'
doi: 10.1073/pnas.1304246110
abstract: Planktonic bacteria dominate surface ocean biomass and influence global
  biogeochemical processes, but remain poorly characterized owing to difficulties
  in cultivation. Using large-scale single cell genomics, we obtained insight into
  the genome content and biogeography of many bacterial lineages inhabiting the surface
  ocean. We found that, compared with existing cultures, natural bacterioplankton
  have smaller genomes, fewer gene duplications, and are depleted in guanine and cytosine,
  noncoding nucleotides, and genes encoding transcription, signal transduction, and
  noncytoplasmic proteins. These findings provide strong evidence that genome streamlining
  and oligotrophy are prevalent features among diverse, free-living bacterioplankton,
  whereas existing laboratory cultures consist primarily of copiotrophs. The apparent
  ubiquity of metabolic specialization and mixotrophy, as predicted from single cell
  genomes, also may contribute to the difficulty in bacterioplankton cultivation.
  Using metagenome fragment recruitment against single cell genomes, we show that
  the global distribution of surface ocean bacterioplankton correlates with temperature
  and latitude and is not limited by dispersal at the time scales required for nucleotide
  substitution to exceed the current operational definition of bacterial species.
  Single cell genomes with highly similar small subunit rRNA gene sequences exhibited
  significant genomic and biogeographic variability, highlighting challenges in the
  interpretation of individual gene surveys and metagenome assemblies in environmental
  microbiology. Our study demonstrates the utility of single cell genomics for gaining
  an improved understanding of the composition and dynamics of natural microbial assemblages.
links:
- name: URL
  url: https://pnas.org/doi/full/10.1073/pnas.1304246110
pages: '11463-11468'
volume: '110'
number: '28'
---
