---
title: 'Metagenomic 16S rDNA Illumina tags are a powerful alternative to amplicon sequencing to explore diversity and structure of microbial communities'
authors:
- Ramiro Logares
- Shinichi Sunagawa
- Guillem Salazar
- Francisco M. Cornejo-Castillo
- Isabel Ferrera
- Hugo Sarmento
- Pascal Hingamp
- Hiroyuki Ogata
- Colomban De Vargas
- Gipsi Lima‐Mendez
- Jeroen Raes
- Julie Poulain
- Olivier Jaillon
- Patrick Wincker
- Stefanie Kandels‐Lewis
- Eric Karsenti
- Peer Bork
- Silvia G. Acinas
date: '2014-09-01'
publishDate: '2024-08-05T15:04:47.129341Z'
publication_types:
- article-journal
publication: '*Environmental Microbiology*'
doi: 10.1111/1462-2920.12250
abstract: Summary  Sequencing of 16S rDNA polymerase chain reaction ( PCR ) amplicons
  is the most common approach for investigating environmental prokaryotic diversity,
  despite the known biases introduced during PCR . Here we show that 16S rDNA fragments
  derived from Illumina‐sequenced environmental metagenomes (   mi tag   s) are a
  powerful alternative to 16S rDNA amplicons for investigating the taxonomic diversity
  and structure of prokaryotic communities. As part of the  T ara   O ceans global
  expedition, marine plankton was sampled in three locations, resulting in 29 subsamples
  for which metagenomes were produced by shotgun Illumina sequencing (ca. 700 Gb).
  For comparative analyses, a subset of samples was also selected for R oche‐454 sequencing
  using both shotgun (   m454 tag   s; 13 metagenomes, ca. 2.4 Gb) and 16S rDNA amplicon
  (   454 tag   s; ca. 0.075 Gb) approaches. Our results indicate that by overcoming
  PCR biases related to amplification and primer mismatch,   mi tag   s may provide
  more realistic estimates of community richness and evenness than amplicon   454
  tag   s. In addition,   mi tag   s can capture expected beta diversity patterns.
  Using   mi tag   s is now economically feasible given the dramatic reduction in
  high‐throughput sequencing costs, having the advantage of retrieving simultaneously
  both taxonomic ( B acteria, A rchaea and E ukarya) and functional information from
  the same microbial community.
links:
- name: URL
  url: https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.12250
pages: '2659-2671'
volume: '16'
number: '9'
---
