---
title: A consensus protocol for the recovery of mercury methylation genes from metagenomes
authors:
- Eric Capo
- Benjamin D. Peterson
- Minjae Kim
- Daniel S. Jones
- Silvia G. Acinas
- Marc Amyot
- Stefan Bertilsson
- Erik Björn
- Moritz Buck
- Claudia Cosio
- Dwayne A. Elias
- Cynthia Gilmour
- Marisol Goñi-Urriza
- Baohua Gu
- Heyu Lin
- Yu-Rong Liu
- Katherine McMahon
- John W. Moreau
- Jarone Pinhassi
- Mircea Podar
- Fernando Puente-Sánchez
- Pablo Sánchez
- Veronika Storck
- Yuya Tada
- Adrien Vigneron
- David A. Walsh
- Marine Vandewalle-Capo
- Andrea G. Bravo
- Caitlin M. Gionfriddo
date: '2023-01-01'
publishDate: '2024-08-08T12:00:46.964078Z'
publication_types:
- article-journal
publication: '*Molecular Ecology Resources*'
doi: 10.1111/1755-0998.13687
abstract: Mercury (Hg) methylation genes (hgcAB) mediate the formation of the toxic
  methylmercury and have been identified from diverse environments, including freshwater
  and marine ecosystems, Arctic permafrost, forest and paddy soils, coal-ash amended
  sediments, chlor-alkali plants discharges and geothermal springs. Here we present
  the first attempt at a standardized protocol for the detection, identification and
  quantification of hgc genes from metagenomes. Our Hg-cycling microorganisms in aquatic
  and terrestrial ecosystems (Hg-MATE) database, a catalogue of hgc genes, provides
  the most accurate information to date on the taxonomic identity and functional/metabolic
  attributes of microorganisms responsible for Hg methylation in the environment.
  Furthermore, we introduce “marky-coco”, a ready-to-use bioinformatic pipeline based
  on de novo single-metagenome assembly, for easy and accurate characterization of
  hgc genes from environmental samples. We compared the recovery of hgc genes from
  environmental metagenomes using the marky-coco pipeline with an approach based on
  coassembly of multiple metagenomes. Our data show similar efficiency in both approaches
  for most environments except those with high diversity (i.e., paddy soils) for which
  a coassembly approach was preferred. Finally, we discuss the definition of true
  hgc genes and methods to normalize hgc gene counts from metagenomes.
tags:
- metagenomics
- bioinformatics
- hg methylation
- hg-MATE
- hgcAB genes
- marky-coco
- mercury
links:
- name: URL
  url: https://onlinelibrary.wiley.com/doi/abs/10.1111/1755-0998.13687
pages: '190-204'
volume: '23'
number: '1'
---
