---
title: 'Getting specific: making taxonomic and ecological sense of large sequencing
  data sets'
authors:
- Ramon Massana
date: '2015-06-01'
publishDate: '2024-08-05T15:04:46.886614Z'
publication_types:
- article-journal
publication: '*Molecular Ecology*'
doi: 10.1111/mec.13210
abstract: Eukaryotic microbes comprise a diverse collection of phototrophic and heterotrophic
  creatures known to play fundamental roles in ecological processes. Some can be identified
  by light microscopy, generally the largest and with conspicuous shapes, while the
  smallest can be counted by epifluorescence microscopy or flow cytometry but remain
  largely unidentified. Microbial diversity studies greatly advanced with the analysis
  of phylogenetic markers sequenced from natural assemblages. Molecular surveys began
  in 1990 targeting marine bacterioplankton (Giovannoni et al .  ) and first approached
  microbial eukaryotes in three studies published in 2001 (Díez et al .  ; López‐García
  et al .  ; Moon‐van der Staay et al .  ). These seminal studies, based on cloning
  and Sanger sequencing the complete 18S rDNA , were critical for obtaining broad
  pictures of microbial diversity in contrasted habitats and for describing novel
  lineages by robust phylogenies, but were limited by the number of sequences obtained.
  So, inventories of species richness in a given sample and community comparisons
  through environmental gradients were very incomplete. These limitations have been
  overcome with the advent of high‐throughput sequencing ( HTS ) methods, initially
  454‐pyrosequencing, today Illumina and soon others to come. In this issue of Molecular
  Ecology , Egge et al . (  ) show a nice example of the use of HTS to study the biodiversity
  and seasonal succession of a particularly important group of marine microbial eukaryotes,
  the haptophytes. Temporal changes were analysed first at the community level, then
  at the clade level, and finally at the lowest rank comparable to species. Interesting
  and useful ecological insights were obtained at each taxonomic scale. Haptophyte
  diversity differed along seasons in a systematic manner, with some species showing
  seasonal preferences and others being always present. Many of these species had
  no correspondence with known species, pointing out the high level of novelty in
  microbial assemblages, only accessible by molecular tools. Moreover, the number
  of species detected was limited, agreeing with a putative scenario of constrained
  evolutionary diversification in free‐living small eukaryotes. This study illustrates
  the potential of HTS to address ecological relevant questions in an accessible way
  by processing large data sets that, nonetheless, need to be treated with a fair
  understanding of their limitations.
links:
- name: URL
  url: https://onlinelibrary.wiley.com/doi/10.1111/mec.13210
pages: '2904-2906'
volume: '24'
number: '12'
---
